go/pserver/client/c/test/test_train.py

```diff
@@ -31,6 +31,13 @@ def main():
 
     # create optimizer of new remote updater to pserver
     optimizer = paddle.optimizer.Momentum(momentum=0, learning_rate=1e-3)
+    optimizer.set_remote_optimizer_config(
```

Comment:
I think we should **not** add a new API to the Python interface. How hard would it be if we keep the current Python interface, but convert the parsed V1 proto to V2 proto internally? (seems you have already write most of the code to do so).